### PR TITLE
Mccalluc/npm test

### PR DESCRIPTION
**Pull request by mccalluc**
Turn JS tests on. The prov vis tests I ported over did not run out of the box... So I tried some global search and replace, which helped, but in the end commented out some tests.

I think getting the JS tests back in CI is important... perhaps it would be better just to delete the prov tests that aren't running than to disable them?

### CODE REVIEW

**Comment by mccalluc** [comment]
Fix #792.

File: test.sh [hunk 1]
```diff
@@ -64,6 +64,13 @@ pytest
 end pytest
 
 
+start npm-test
+cd context
+npm test
```
Comment (john-conroy) [inline test.sh:69]:
```suggestion
npm run test
```

**Comment by mccalluc** [comment]
@john-conroy : `node_modules/jest/bin/jest.js test` passes locally, but `npm run test` doesn't: Errors look like:
```
    SyntaxError: Invalid or unexpected token
        at compileFunction (<anonymous>)
      1 | import { createGlobalStyle } from 'styled-components';
      2 | 
    > 3 | import InterVariable from 'typeface-inter/Inter (web)/Inter.var.woff2';
```
Something different about the transpiling process between those two commands? If it's not obvious to you, I would argue for just changing to `jest.js test` for now, to have something in place, and then loop back later to understand why the preferred method doesn't work.

**Comment by john-conroy** [comment]
We probably need a mock. So use `jest.js` test for now and I can triage this later.

**Pull request merged**